workflow/doc.go

```diff
@@ -286,14 +357,14 @@ pattern, extra care needs to be taken to ensure the child workflow is started be
 Error Handling
 
 Activities and child workflows can fail. You could handle errors differently based on different error cases. If the
-activity returns an error as errors.New() or fmt.Errorf(), those errors will be converted to error.GenericError. If the
-activity returns an error as error.NewCustomError("err-reason", details), that error will be converted to
-*error.CustomError. There are other types of errors like error.TimeoutError, error.CanceledError and error.PanicError.
+activity returns an error as errors.New() or fmt.Errorf(), those errors will be converted to workflow.GenericError. If the
+activity returns an error as workflow.NewCustomError("err-reason", details), that error will be converted to
+*workflow.CustomError. There are other types of errors like workflow.TimeoutError, workflow.CanceledError and workflow.PanicError.
 So the error handling code would look like:
 
 	err := workflow.ExecuteActivity(ctx, YourActivityFunc).Get(ctx, nil)
 	switch err := err.(type) {
-	case *error.CustomError:
```

Comment:
we have no `error` package ¯\\\_(ツ)\_/¯

Reply:
even if we have, I think using `workflow.XyzError is more friendly than `error.XyzError` because there will be a lot of `error` pkg loaded into the IDE.

Reply:
yeah, "error" and "errors" are kinda risky in that way.  lots of chances for collisions -> annoying manual aliases.